package.json

```diff
@@ -51,7 +51,7 @@
     "deploy:npm": "nx affected --target=deploy --parallel=1 --skipCommitTypes=docs,ci,chore,test"
   },
   "author": "RudderStack",
-  "license": "MIT",
+  "license": "SEE LICENSE IN LICENSE.md",
```

Comment:
```suggestion
  "license": "ELv2",
```

Please make similar changes accordingly

Reply:
Done.

Reply:
The correct SPDX identifier is - `Elastic-2.0`
Reference - https://spdx.org/licenses/

Reply:
Addressed.